### PR TITLE
Black formatting fixed

### PR DESCRIPTION
**Pull request by rhaps0dy**
Add black formatting check in CircleCI and make it pass.

Merge after #24 . Fixes part of #20

### CODE REVIEW

File: pyproject.toml
Comment (rhaps0dy) [inline pyproject.toml:1]: check this

**Comment by rhaps0dy** [comment]
I think this is also ready to merge right after #24

**Comment by rhaps0dy** [comment]
Can I fix? If so please approve @nikihowe

**Review by nikihowe — APPROVED** [review]

**Pull request merged**
